README.md

```diff
@@ -17,191 +19,110 @@ We've collected and crafted a rich collection of spinners animated with css whic
 Special thanks to [@martinvd](https://github.com/martinvd) for his outstanding codepens :)
 
 ## Demo & Documentation
+
 [View demo](http://epic-spinners.epicmax.co) to see vue.js components usage examples and html/css source code
 
 ## Installation
+
 ```
 npm install --save epic-spinners
 ```
 
 ## Usage
-Vue.js usage example 
 
-``` vue
+Vue.js usage example
+
+```ts
 <template>
   <div id="app">
-     <atom-spinner
-          :animation-duration="1000"
-          :size="60"
-          :color="'#ff1d5e'"
-     />
+    <atom-spinner :animation-duration="1000" :size="60" color="#ff1d5e" />
   </div>
 </template>
 
 <script>
-  // To use minified css and js files instead of .vue single file components:
```

Comment:
Are you sure it's getting tree shaked without file import?

Reply:
After setting `sideEffects: ['*.css']` tree-shaking works in production mode without file import